reference-implementation/unittests/tosa.cpp

```diff
@@ -27,62 +28,192 @@ const float EPSILON = 5e-4;
 
 // Unary elementwise ops
 TEST(tosa, clamp) {
-  Tensor<float, 2, 1> t0{-1.5f, 5.0f};
-  float min0 = -1.0f;
-  float max0 = 3.0f;
-  Tensor<float, 2, 1> expected_result0{-1.0, 3.0f};
-  Tensor<float, 2, 1> s0 = tosa::clamp(t0, min0, max0);
-
-  EXPECT_THAT(s0, Pointwise(FloatEq(), expected_result0));
-
-  Tensor<int64_t, 4, 2, 1> t1{-2, 2, -2, 3, 4, -5, 5, 5};
-  int64_t min1 = 1;
-  int64_t max1 = 3;
-  Tensor<int64_t, 4, 2, 1> expected_result1{1, 2, 1, 3, 3, 1, 3, 3};
-  Tensor<int64_t, 4, 2, 1> s1 = tosa::clamp(t1, min1, max1);
-  EXPECT_THAT(s1, Pointwise(Eq(), expected_result1));
+  {
+    Tensor0D<int32_t> operand{-1};
+    int32_t min_value = -3;
```

Comment:
Should we generally define for our internal style guide that parameter variables need to be defined in the same order that they appear in in the function signature?
Otherwise reordering might be nicer visually, but that's a nitpick.

Reply:
I think having the arguments in the same order as in the signature is a good rule. Which way it looks nicer is quite subjective.

Reply:
Let's order the arguments according to the function signature :)